person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
```

Comment:
## OPTIONAL 

- [ ] It would be better to make your id a random number from 1 - 1000 instead of 100, this way you are sure all id's are unique 👍🏿